### PR TITLE
Helpers class

### PR DESCRIPTION
**Pull request by berwinlan**
Write and test function for making an API call to Google Maps Distance Matrix

### CODE REVIEW

File: testing/unit_tests/test_helpers.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    midpoints = ['Charleston%2CMA', 'Concord%2CMA', 'Sudbury%2CMA', 'Westwood%2CMA', 'Arlington%2CMA', 'Needham%2CMA']
+
+    # Make API call to Google Distance Matrix
+    data = call_api(start, end, midpoints)
+
+    # Load sample data
+    with open('testing/integration_tests/test_response.json') as f:
+        sample_data = json.load(f)
+
+    # Traffic data will vary
```
Comment (benmor20) [inline testing/unit_tests/test_helpers.py]: `assert` that the traffic data exists (without asserting specific value)

**Review by benmor20 — CHANGES_REQUESTED** [review]
Overall looks good, but one or two extra things in the tests would be nice

**Review by benmor20 — APPROVED** [review]
Looks good!

File: src/utils/helpers.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        (JSON) Response in JSON format
+    '''
+    load_dotenv()
+    API_KEY = os.getenv('API_KEY')
+
+    # Join midpoints into string
+    midpoint_str = '%7C'.join(midpoints)
+
+    # Create URL
+    url = f"https://maps.googleapis.com/maps/api/distancematrix/json?origins={start}%7C{midpoint_str}&destinations={end}%7C{midpoint_str}&departure_time=now&key={API_KEY}"
```
Comment (hnvakil) [inline src/utils/helpers.py]: Might be better practice to have the url components (like up to /json for one) as variables then smoosh them all together from what I've seen people in industry do in the past but not a serious problem

File: src/utils/helpers.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+    # Make HTTP request
+    response = requests.request("GET", url, headers={}, data={})
+
+    print(response.text)
+    print(API_KEY)
```
Comment (hnvakil) [inline src/utils/helpers.py]: why are we printing api key?

Comment (berwinlan) [reply src/utils/helpers.py]: good question! we should not be 😌

File: src/utils/helpers.py (hunk 2 above)
Comment (berwinlan) [reply src/utils/helpers.py]: smoooooooooooooooooshed

File: testing/unit_tests/test_helpers.py (hunk 1 above)
Comment (berwinlan) [reply testing/unit_tests/test_helpers.py]: Done - asserted that `['rows']` was non-empty

**Pull request merged**